src/main/java/duke/parser/Command.java

```diff
@@ -0,0 +1,136 @@
+package duke.parser;
+
+import duke.task.TaskList;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.ToDoTask;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+import java.util.Scanner;
+
+import java.util.List;
+
+import java.util.stream.Collectors;
+
+public class Command {
+    Scanner terms;
+
+    Command(Scanner sc) {
+        this.terms = sc;
+    }
+
+    /**
+     * Add new user task into task list
+     *
+     * @param first   = task type specified by user
+     * @param terms   = subsequent terms specified by to define the task
+     * @param tasks   = task list
+     * @param ui      = ui for responding to user after adding the new task
+     * @param storage = save to local file after every changes to task list
+     * @return task successfully added?
+     */
+    static boolean addTask(String first, Scanner terms, TaskList tasks, Ui ui, Storage storage) {

```

Comment:
Why did you choose to have these methods pertaining to TaskList and Task in the Command class rather than say, the TaskList/Task class?